db/migrate/20150903081526_add_foreman_tasks_bookmarks.rb

```diff
@@ -0,0 +1,25 @@
+class AddForemanTasksBookmarks < ActiveRecord::Migration
+  def up
+    Bookmark.where(:name => "pending", 
+                   :query => "result = pending", 
+                   :controller => "foreman_tasks_tasks",
+                   :public => true).first_or_create
+    Bookmark.where(:name => "paused", 
```

Comment:
Seeds would be a better place for this than migrations, see https://github.com/theforeman/foreman/blob/develop/db/seeds.d/15-bookmarks.rb